src/Orleans.Persistence.Cassandra/Storage/CassandraGrainStorage.cs

```diff
@@ -310,6 +310,13 @@ private async Task Init(CancellationToken cancellationToken)
                                   .AddContactPoints(cassandraOptions.ContactPoints.Split(','))
                                   .Build();
 
+                // when there are less nodes, the driver throws Cassandra.UnavailableExeption with Quorum
+                if (cassandraOptions.ReplicationFactor < 3)
+                {
+                    DefaultConsistencyLevel = ConsistencyLevel.One;
+                    DefaultConsistencyQueryOptions.SetConsistencyLevel(DefaultConsistencyLevel);
```

Comment:
This is updating a global static field. Better to have an instance field instead